packages/libs/web-common/src/components/reporters/SequenceFormFactory.jsx

```diff
@@ -90,7 +90,11 @@ const createSequenceForm = (
       includeSubmit,
       viewFilters,
       updateViewFilters,
+      resultType,
     } = props;
+    const shouldRenderTranscriptFilter =
+      resultType?.step?.recordClassName &&
+      resultType?.step?.recordClassName === 'transcript';
```

Comment:
You should use https://github.com/VEuPathDB/web-monorepo/blob/b6b240468f82ccc6fd2051b1ff2a72782efb7857/packages/libs/wdk-client/src/Utils/WdkResult.ts#L47-L48 to get the `recordClassName` of the `resultType`:

```ts
const resultTypeDetailsResult = useWdkService(wdkService => getResultTypeDetails(wdkService, resultType));
const shouldRenderTranscriptFilter = resultTypeDetailsResult.value?.recordClassName === 'transcript';
```

Reply:
I'm getting an infinite loop with this approach. Any idea why?